README.md

```diff
@@ -88,11 +88,12 @@ For recording, the executable needs to be compiled with the `-pg`
 (or `-finstrument-functions`) option which generates profiling code
 (calling mcount or __cyg_profile_func_enter/exit) for each function.
 
-Note that, there's an experimental support for dynamic tracing on x86_64
-which doesn't require such (re-)compilations.  Also recent compilers have
-some options to help uftrace to reduce tracing overhead with similar way
-(although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+Note that, there's an experimental support for dynamic tracing on 
+x86_64 and AArch64(ARM64) which doesn't require such (re-)compilations.  
+Also recent compilers have some options to help uftrace 
+to reduce tracing overhead with similar way
+(although it still needs recompilation of your program).  
+Please see [doc/uftrace-record.md](doc/uftrace-record.md) file.
```

Comment:
Please remove all the white spaces at the end of lines. They are shown as red blocks in the diff output.
<img width="687" alt="Screen Shot 2019-11-11 at 11 22 04 PM" src="https://user-images.githubusercontent.com/19642328/68594160-48dc4c00-04da-11ea-9a09-592ae8baf220.png">